### DISCUSSION TITLE
Memcached meta-data error

### DISCUSSION THREAD

**Discussion by quasipickle** [phalcon/cphalcon#16225] [Q&A] [answered]
I'm not sure if this is a bug or misconfiguration - I'll re-post as a bug if necessary.

I'm trying to set up a `memcached` cache for my model meta-data.  When I do so, I get this notice:
```
PHP Notice: Cannot use a scalar value as an array in phalcon/Mvc/Model/MetaData.zep on line 630 
in [trimmed for brevity]AlbumController.php:14
```
And a thrown exception saying `The meta-data is invalid or is corrupt` from `phalcon/Mvc/Model/MetaData.zep:415`

My cache service is set up like this:
```php
$Container->set("modelsMetadata", function () {
    $serializerFactory = new Phalcon\Storage\SerializerFactory();
    $adapterFactory = new Phalcon\Cache\AdapterFactory($serializerFactory);
    $options = [
        "servers" => [
            0 => [
                "host" => "127.0.0.1",
                "port" => 11211,
                "weight" => 1
            ]
        ],
        "lifetime" => 86400,
        "prefix" => "pl-2022-11-24-0"
    ];

    return new Phalcon\Mvc\Model\Metadata\Libmemcached($adapterFactory, $options);
});
```
The only thing I have in my models' `initialize()` is setting up relationships.

**System:**
PHP: 8.1.12
Ubuntu: 20.04 (in WSL2 in Windows 10)
Phalcon: 5.1.1

Have I misconfigured something or is this a bug?

**Comment by niden** [comment, marked as answer]
Transferring this to an issue.
